skills/hook-kit/resources/bash-guard.py

```
@@ -946,12 +946,14 @@ def hard(reason: str) -> tuple[int, list[str], list[str]]:
 
     # ── Phase 2 ──
     if re.search(r"git\s+commit", command) and "--amend" not in command:
+        staged_files: list[str] = []
         try:
             r = subprocess.run(
                 ["git", "diff", "--cached", "--name-only"],
```

Ruff (0.16.3)

[error] Starting a process with a partial executable path

(S607)